code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -462,8 +471,21 @@ This is here to make the tiles around the station mininuke change when it's arme
 			var/obj/item/weapon/storage/S = loc
 			S.remove_from_storage(src, targetturf)
 		forceMove(targetturf) //move the disc, so ghosts remain orbitting it even if it's "destroyed"
-		message_admins("[src] has been destroyed in ([diskturf ? "[diskturf.x], [diskturf.y] ,[diskturf.z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[diskturf.x];Y=[diskturf.y];Z=[diskturf.z]'>JMP</a>":"nonexistent location"]). Moving it to ([targetturf.x], [targetturf.y], [targetturf.z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[targetturf.x];Y=[targetturf.y];Z=[targetturf.z]'>JMP</a>).")
-		log_game("[src] has been destroyed in ([diskturf ? "[diskturf.x], [diskturf.y] ,[diskturf.z]":"nonexistent location"]). Moving it to ([targetturf.x], [targetturf.y], [targetturf.z]).")
+		return targetturf
 	else
 		throw EXCEPTION("Unable to find a blobstart landmark")
+
+/obj/item/weapon/disk/nuclear/Destroy(force)
+	var/turf/diskturf = get_turf(src)
+
+	if(force)
```

Comment:
The force argument was present before this PR during my changes to the GC that allowed for normally immortal items to be force deleted. It shouldn't happen in normal gameplay.